cmd/http-server/startcmd/start.go

```diff
@@ -169,6 +171,19 @@ const (
 	oidcCallbackURLEnvKey = "HTTP_SERVER_OIDC_CALLBACK"
 )
 
+// Keys.
+const (
+	sessionCookieAuthKeyFlagName  = "cookie-auth-key"
+	sessionCookieAuthKeyFlagUsage = "Path to the pem-encoded 32-byte key to use to authenticate session cookies." +
+		" Alternatively, this can be set with the following environment variable: " + sessionCookieAuthKeyEnvKey
+	sessionCookieAuthKeyEnvKey = "HTTP_SERVER_COOKIE_AUTH_KEY"
+
+	sessionCookieEncKeyFlagName  = "cookie-enc-key"
+	sessionCookieEncKeyFlagUsage = "Path to the pem-encoded 32-byte key to use to encrypt session cookies." +
+		" Alternatively, this can be set with the following environment variable: " + sessionCookieEncKeyEnvKey
+	sessionCookieEncKeyEnvKey = "HTTP_SERVER_COOKIE_ENC_KEY"
+)
+
 var logger = log.New("edge-agent/http-server")
```

Comment:
unrelated to this PR, but i think we should have a more specific binary name that makes it clear that this is the web agent / wallet server.

Reply:
#397